roles/jboss_eap/tasks/main.yml

```diff
@@ -115,3 +115,88 @@
       ignore_errors: yes
       become: true
       when: '"jboss.eap.init-files" in facts_to_collect'
+
+# Tasks that can locate a KARAF_HOME directory
```

Comment:
Sorry for the confusion, just wanted the role to be run in a different order. Not combined with the eap role.

Reply:
My mistake! I just pushed another change which I think is what you are looking for.